README.md

```diff
@@ -40,7 +55,7 @@ make dev
 
 This should run `npm install` and then open [the `sites` root file](./src/sites/index.html) at http://localhost:3333/ automatically. Any source code changes will refresh the browser. We don't yet have a development guide, so you'll have to search the code to find where features are implemented.
 
-The core library code for `cubing.js` is in [`src/cubing`](./src/cubing/) To test changes, use:
+The core library code for `cubing.js` is in [src/cubing](./src/cubing/) To test changes, use:
```

Comment:
Please keep the code formatting! :-D

(I also just noticed I missed a period right after.)

```suggestion
The core library code for `cubing.js` is in [`src/cubing`](./src/cubing/). To test changes, use:
```